socorro/external/postgresql/connection_context.py

```diff
@@ -162,14 +162,15 @@ def is_operational_exception(self, msg):
         operational exception "labelled" wrong by the psycopg2 code error
         handler.
         """
-        if msg.pgerror in ('SSL SYSCALL error: EOF detected', ):
-            # Ideally we'd like to check against msg.pgcode values
+        message = exp.args[0]
+        if message in ('SSL SYSCALL error: EOF detected', ):
+            # Ideally we'd like to check against exp.pgcode values
             # but certain odd ProgrammingError exceptions don't have
             # pgcodes so we have to rely on reading the pgerror :(
             return True
 
         if (
-            isinstance(msg, psycopg2.OperationalError) and msg.pgerror not in (
+            isinstance(exp, psycopg2.OperationalError) and message not in (
```

Comment:
Now I'm guilty of superficial comments about style in a code review...   This if statement would be easier to read at a glance, if it were written as:
```python
        if (
            isinstance(exp, psycopg2.OperationalError)
            and message not in ('out of memory', )
        ):
```